roles/common/tasks/pod_tests.yml

```
@@ -0,0 +1,20 @@
+---
+
+- name: Get Pod Instance "{{ pod_status_str }}" 
+  ansible.builtin.shell:
+    cmd: |
+      oc get pods -n  "{{ pod_nspace }}" | grep "{{ item }}" | grep "{{ pod_status_str }}" | awk '{print $1;}'
```

If this fails, there is no indication until the next task.
There is also no helpful information to see what might have caused the error.

The next task will fail if there is nothing returned from here.
```suggestion
      oc get pods -n  "{{ pod_nspace }}" | grep "{{ item }}" | grep "{{ pod_status_str }}" | awk '{print $1;}'
    failed_when:
        - podinstance.stdout_lines | length == 0
```